build.sbt

```diff
@@ -616,18 +667,17 @@ lazy val test = project
     fork in IntegrationTest := true,
     javaOptions in IntegrationTest += "-Xmx1G",
     testFrameworks += new TestFramework("scala.tools.partest.sbt.Framework"),
-    testOptions in IntegrationTest += Tests.Setup( () => root.base.getAbsolutePath + "/pull-binary-libs.sh" ! ),
```

Comment:
yeah, so as expected: after this PR, `partest test/files/pos/cycle-jsoup.scala` fails until `pull-binary-libs.sh` is run manually. i think we should find a way to fix this, it's too confusing for newcomers.